packages/nouns-api/src/services/ideas.ts

```diff
@@ -25,6 +27,15 @@ const calculateVotes = (votes: any) => {
   return count;
 };
 
+const calculateConsensus = (idea: Idea) => {
+  if (!idea.tokenSupplyOnCreate) {
+    return undefined;
```

Comment:
If this is undefined on the client (pre-existing ideas) we just won't show the tag. We can look into backfilling this data for old ideas at a later date if it's required.